odbc/src/api/functions.rs

```diff
@@ -833,7 +833,7 @@ pub unsafe extern "C" fn SQLDisconnect(connection_handle: HDbc) -> SqlReturn {
 fn sql_driver_connect(conn: &Connection, odbc_uri_string: &str) -> Result<MongoConnection> {
     let mut odbc_uri = ODBCUri::new(odbc_uri_string)?;
     let mongo_uri = odbc_uri.remove_to_mongo_uri()?;
-    let auth_src = odbc_uri.remove_or_else(|| "admin", &["auth_src"]);
```

Comment:
If we are removing the auth_src argument, we should delete this line.

Reply:
That's right